testing/sdk_metrics.py

```diff
@@ -1,6 +1,11 @@
 '''
 Utilities relating to verifying the metrics functionality as reported
 by the DC/OS metrics component.
+
+************************************************************************
+FOR THE TIME BEING WHATEVER MODIFICATIONS ARE APPLIED TO THIS FILE
```

Comment:
This is the case with all of `testing/` though? Is there anything special with this case?

Reply:
Included this in all of `/testing/`